drivers/crypto/crypto_ataes132a.c

```diff
@@ -24,6 +21,10 @@
 /* ATAES132A can store up to 16 different crypto keys */
 #define CRYPTO_MAX_SESSION 16
 
+#define LOG_LEVEL CONFIG_CRYPTO_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ataes132a);
```

Comment:
Here you are registering a driver-specific logging module, however in other subsystems all drivers had the same name in logger (for example "adc").

IMHO we should register every driver using it's name (like it is done here) as this allows for fine grained log control over console as well as does not produce conflicts if 2 drivers form same subsystem are simultaneously compiled-in.

Reply:
most systems would have only one peripheral from each type so the use of the subsystem here is cleaner IMO, but I am fine changing this and did for most of the drivers. Some become very long, like 'clock_control_xxxxx_xxxxx', IMO this is not nice in a log...